etc/profile-m-z/mpg123.profile

```diff
@@ -23,19 +23,24 @@ include whitelist-var-common.inc
 apparmor
 caps.drop all
 netfilter
+no3d
+nodvd
```

Comment:
nodvd also includes (audio) cd.

Reply:
@rusty-snake Indeed. I assumed that audio CD's are not in a compatible format for mpg123 to process directly. So a decoded WAV or MP3 file would reside on the regular filesystem instead. But I forgot archived audio files. Can't test due to having exchanged my laptop's CD/DVD drive for an extra SSD years ago. I'll remove the nodvd, it's a minor change anyway.